src/modules/user/auth.helper.ts

```diff
@@ -0,0 +1,84 @@
+import { Injectable, NotFoundException } from '@nestjs/common';
```

Comment:
@gerimujo496 Usually in helper files we define simple utilities (e.g. generateToken). All other methods belong to the service layer (you have communication with the data access layer, you can do it directly in the service, no need for separate helpers